enterprise/frontend/src/metabase-enterprise/llm_autodescription/LLMSuggestQuestionInfo.tsx

```diff
@@ -2,18 +2,30 @@ import { useState } from "react";
 import { useAsync } from "react-use";
 import { t } from "ttag";
 
+import { canonicalCollectionId } from "metabase/collections/utils";
 import { useSetting } from "metabase/common/hooks";
 import { color } from "metabase/lib/colors";
+import { useSelector } from "metabase/lib/redux";
 import type { LLMIndicatorProps } from "metabase/plugins/types";
+import { getSubmittableQuestion } from "metabase/query_builder/selectors";
 import { Button, Icon, Tooltip } from "metabase/ui";
 import { AutoDescribeApi } from "metabase-enterprise/services";
-
 import "./loading.css";
+import type Question from "metabase-lib/v1/Question";
 
 export const LLMSuggestQuestionInfo = ({
-  question,
+  question: initialQuestion,
+  initialCollectionId,
   onAccept,
 }: LLMIndicatorProps) => {
+  const collectionId = canonicalCollectionId(initialCollectionId);
+  const questionWithCollectionId: Question =
+    initialQuestion.setCollectionId(collectionId);
```

Comment:
Is this because the `question` from `useSelector` might change later, so we make sure the `questionWithCollectionId` did not change?

Reply:
It was a simpler reason - we calculate these values in `SaveQuestionModal` but the values just weren't being used anywhere else, so I figured it would make more sense to have this component do that logic